spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -165,6 +166,9 @@
       given_there_is_a_scheduled_step_by_step_page
       and_I_visit_the_publish_or_delete_page
       then_I_see_an_unschedule_button
+      and_I_click_the_unschedule_button
+      then_I_should_see "has been unscheduled"
+      and_the_step_by_step_should_have_the_status "Draft"
```

Comment:
Is it worth having another test (possibly in `spec/models/step_by_step_page_spec.rb`) that makes sure a step by step has status "Unpublished changes" -> "Scheduled" -> "Unpublished changes" (in case we've accidentally made it change back to "Draft").

Reply:
I think it's a good idea to test that the status is being updated appropriately, but that should be triggered from a controller test, rather than a model test.

Reply:
Added 2 test to the controller: 
- Draft -> Scheduled -> Draft
- Unpublished changes -> Scheduled -> Unpublished changes